backend/src/auth/auth.guard.ts

```diff
@@ -96,4 +97,12 @@ export class AuthGuard implements CanActivate {
       request['roles'] = [];
     }
   }
+
+  setRequestUser(payload: JwtPayload, request: Request): void {
+    if (payload.given_name && payload.family_name) {
+      request['username'] = `${payload.given_name} ${payload.family_name}`;
+    } else {
+      request['username'] = "";
```

Comment:
I thought an empty string made the most sense since we're displaying this on the FE. I don't want an error in the case of null. I think most every IDIR should have this and all auth users should have idir.

Reply:
Fair enough on the errors, I would've thought an FE check would be what made sense. However, this should be okay for now!